packages/model-viewer/src/three-components/ARRenderer.ts

```diff
@@ -418,8 +418,10 @@ export class ARRenderer extends EventDispatcher {
       const scale = view.recommendedViewportScale;
       view.requestViewportScale(Math.max(scale, MIN_VIEWPORT_SCALE));
     }
-    const layer = this.currentSession!.renderState.baseLayer;
-    const viewport = layer!.getViewport(view)!;
+    const layer = xr.getBaseLayer();
```

Comment:
Looks like we need to null-check `layer`, based on the CI failure. But the weirder part is that it's failing on Chrome Headless, which I can't imagine supports WebAR. I thought that would fail back in `supportsSession`. Any idea what's going on?

Reply:
No, I don't. Is this running a static analysis?

Reply:
No, just regular JS tests in a headless browser. If you go to `packages/model-viewer` and run `npm test` locally I wonder if it'll repro? Maybe only on Chrome Linux though...

Reply:
I tried running it locally but it didn't work. I suspect those ports are blocked on my machine.